### PR TITLE
Add new markets to a new supermarket on Sepolia

### PR DESCRIPTION
**Pull request by leomassazza**

### CODE REVIEW

File: omnibus-base-sepolia-andromeda.toml [hunk 1]
```diff
@@ -153,6 +153,171 @@ include = [
     "tomls/omnibus-base-sepolia-andromeda/perps/axl-invokes.toml",
     "tomls/oracles/pyth-axl.toml",
     "tomls/omnibus-base-sepolia-andromeda/perps/axl.toml",
+
+    # Second super market for low cap assets
+    # AAVE
+    "tomls/omnibus-base-sepolia-andromeda/perps/aave-invokes.toml",
```
[the remark this answers is not in the document]
Comment (leomassazza) [reply omnibus-base-sepolia-andromeda.toml:161]: didn't find a way to do it for this kind of includes

File: .circleci/config.yml [hunk 2]
```diff
@@ -248,7 +248,7 @@ jobs:
           name: "Build new package upgrading from << parameters.package >>@<< parameters.preset >> and keep RPC running"
           background: true
           command: |
-            yarn cannon build << parameters.toml >> \
+            NODE_OPTIONS=--max_old_space_size=8192 yarn cannon build << parameters.toml >> \
```
Comment (leomassazza) [inline .circleci/config.yml:251]: @noisekit please, confirm this is correct or is a better way to do it.

Comment (noisekit) [reply .circleci/config.yml:251]: I think it is fine. But surprised this is necessary. Why do we need it?

**Comment by leomassazza** [comment]
The build release part is so long that fails to be done with JavaScript heap memory After this release we might be able to remove it. On 11 Jun 2024, at 20:56, Noisekit ***@***.***> wrote:﻿
@noisekit commented on this pull request.



In .circleci/config.yml:
> @@ -248,7 +248,7 @@ jobs:
           name: "Build new package upgrading from << parameters.package >>@<< parameters.preset >> and keep RPC running"
           background: true
           command: |
-            yarn cannon build << parameters.toml >> \
+            NODE_OPTIONS=--max_old_space_size=8192 yarn cannon build << parameters.toml >> \

I think it is fine. But surprised this is necessary. Why do we need it?

**Review by noahlitvin — APPROVED** [review]

**Review by noisekit — APPROVED** [review]

**Pull request merged**